src/components/tl/ucp/tl_ucp.c

```diff
@@ -175,6 +175,32 @@ static ucs_config_field_t ucc_tl_ucp_context_config_table[] = {
      ucc_offsetof(ucc_tl_ucp_context_config_t, pre_reg_mem),
      UCC_CONFIG_TYPE_UINT},
 
+    {"SERVICE_WORKER", "0",
+     "If set to 0, uses the same worker for collectives and "
+     "service. If not, creates a special worker for service collectives "
+     "for which UCX_TL and UCX_NET_DEVICES are configured by the variables "
+     "UCC_TL_UCP_SERVICE_TLS and UCC_TL_UCP_SERVICE_NET_DEVICES respectively",
+     ucc_offsetof(ucc_tl_ucp_context_config_t, service_worker),
+     UCC_CONFIG_TYPE_UINT},
```

Comment:
UCC_CONFIG_TYPE_UINT -> UCC_CONFIG_TYPE_BOOL (the variable should get "y" or "n")

Reply:
ok